src/api/kube.ts

```diff
@@ -1404,6 +1414,52 @@ export class KubeHelper {
     }
   }
 
+  async getOAuthClientAuthorizations(name: string): Promise<string[]> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const { body } = await customObjectsApi.listClusterCustomObject('oauth.openshift.io', 'v1', 'oauthclientauthorizations')
+
+      if (!body.items) {
+        return []
+      }
+      const oauthClientAuthorizations = body.items as any[]
+      return oauthClientAuthorizations.filter(o => o.metadata.name && o.clientName === name)
+    } catch {
+      return []
+    }
+  }
+
+  async deleteOAuthClientAuthorizations(oAuthClientAuthorizations: any[]): Promise<boolean> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const options = new V1DeleteOptions()
+      oAuthClientAuthorizations.forEach(e => customObjectsApi.deleteClusterCustomObject('oauth.openshift.io', 'v1', 'oauthclientauthorizations', e.metadata.name, options))
+      return true
+    } catch (e) {
+      throw this.wrapK8sClientError(e)
+    }
+  }
+
+  async consoleLinkExists(name: string): Promise<boolean> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const { body } = await customObjectsApi.getNamespacedCustomObject('console.openshift.io', 'v1', '' , 'consolelinks', name)
+      return this.compare(body, name)
+    } catch {
+      return false
+    }
+  }
+
+  async deleteConsoleLink(name: string) {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const options = new V1DeleteOptions()
+      await customObjectsApi.deleteNamespacedCustomObject('console.openshift.io', 'v1', '' , 'consolelinks', name, options)
```

Comment:
Is it namespaced or cluster scope object?

Reply:
is cluster scope object... my mistake